src/docs/reference/support-policy.md

```diff
@@ -60,6 +60,37 @@ In the example above, when Version D is released, Version C enters maintenance m
 After April 1st 2022, Version C will be in extended support until July 1st, 2022, six months after the release of
 Version D.
 
+## TypeScript Support
+
+Stencil acts as a compiler for a project's web components, and works closely with the TypeScript compiler to transform
+TSX to vanilla JavaScript. To ensure compatibility between the two, Stencil takes an opinionated stance on which version
+of the TypeScript compiler that must be used.
+
+Stencil includes a recent copy of the TypeScript compiler in its distributable* to guarantee this compatability. 
```

Comment:
Typo:
```suggestion
Stencil includes a recent copy of the TypeScript compiler in its distributable* to guarantee this compatibility. 
```

I don't know if the site supports it, but this could be a Markdown footnote:

```suggestion
Stencil includes a recent copy of the TypeScript compiler in its distributable[^1] to guarantee this compatibility. 
```

Reply:
Typo fixed in https://github.com/ionic-team/stencil-site/pull/843/commits/665b87314bc4bde8458d84f41a159c1245616e82